languages/apex/generic/Parse_apex_tree_sitter.ml

```diff
@@ -5980,41 +5982,50 @@ and sobject_return (env : env) ((v1, v2) : CST.sobject_return) : raw =
     match v2 with
     | Some (v1, v2, v3, v4, v5, v6, v7, v8) ->
         let v1 = (* "(" *) str env v1 in
-        let v2 = selected_fields env v2 in
+        let v2 =
+          R.List
+            (List.map
+               (fun e -> R.Any (G.E e))
+               (selected_fields env v2))
+        in
```

Comment:
A bit of a naive question: if that stuff is inside Raw (last line of function), does this have any effect?